src/v/cloud_storage_clients/abs_client.cc

```diff
@@ -663,4 +683,47 @@ ss::future<abs_client::list_bucket_result> abs_client::do_list_objects(
       });
 }
 
+ss::future<result<abs_client::storage_account_info, error_outcome>>
+abs_client::get_account_info(ss::lowres_clock::duration timeout) {
+    return send_request(
+      do_get_account_info(timeout), bucket_name{""}, object_key{""});
```

Comment:
Unrelated, but is `bucket_name` intentionally unused in `send_request()`?

Reply:
I think it's a bug tbh. I spotted that too, but forgot to go back and clean that up. I'll push an extra commit.

Reply:
Fixed